### ISSUE TITLE
warning: process `cjdroute' used the deprecated sysctl system call with 1.40.6.

### ISSUE THREAD

**Issue by kpcyrd** [hyperboria/cjdns#70]
A bug has been reported to the arch package:

```
[ 82.480304] warning: process `cjdroute' used the deprecated sysctl system call with 1.40.6.
```

This is triggered by `crypto/random/seed/LinuxRandomUuidSysctlRandomSeed.c`.

```
int mib[] = { CTL_KERN, KERN_RANDOM, RANDOM_UUID };
[...]
if (sysctl(mib, 3, output, &sixteen, NULL, 0)
```

Reference: https://lwn.net/Articles/247237/

**Comment by Kubuxu** [comment]
I can take that one.

**Comment by kpcyrd** [comment]
Fixed in cjdelisle/cjdns#857. Thanks @Kubuxu
